### ISSUE TITLE
Use CDK to define service control policy

### ISSUE THREAD

**Issue by 0xjjoyy** [org-formation/org-formation-cli#128]
Hi,

I'd like to be able to use the AWS CDK PolicyStatement to define the service control policy document and statements. Then use org-formation to deploy the service control policy to the desired targets.

Can you help describe or document how I can integrate org-formation with CDK with keeping the policy definition using CDK constructs and use org-formation?

Thanks

**Comment by eduardomourar** [comment]
@0xjjoyy, you should register this resource type (https://github.com/org-formation/aws-resource-providers/blob/master/organizations/policy/README.md) instead. This way you can just deploy using CDK with a L1 construct.

**Comment by 0xjjoyy** [comment]
@eduardomourar any plans to put the [organizations policy type](https://github.com/org-formation/aws-resource-providers/blob/master/organizations/policy/docs/README.md) into the public CloudFormation registry

https://aws.amazon.com/blogs/aws/introducing-a-public-registry-for-aws-cloudformation/

https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/publish-extension.html

**Comment by eduardomourar** [comment]
Yes, I will start working on it next month. It just might be a slow process to migrate every resource type there.

**Comment by 0xjjoyy** [comment]
Cool! My vote would be to start with the org policy :)

**Comment by 0xjjoyy** [comment]
Hi, just checking to see if you are moving forward with migrating this to the public CloudFormation registry?

**Comment by eduardomourar** [comment]
Unfortunately, I have not had the time to go deep into this, but it is still on our radar.
